releases/nightly-build/compose-files/docker-compose-nexus-arm64.yml

```diff
@@ -0,0 +1,559 @@
+    ports:
+      - "48061:48061"
+    container_name: edgex-support-logging
+    hostname: edgex-support-logging
+    networks:
+      - edgex-network
+    volumes:
+      - log-data:/edgex/logs
+      - vault-config:/vault/config
+      - /tmp/edgex/secrets/ca/ca.pem:/tmp/edgex/secrets/ca/ca.pem:ro
+    depends_on:
+      - config-seed
+      - mongo
+      - volume
+
+  system:
+    image: nexus3.edgexfoundry.org:10004/docker-sys-mgmt-agent-go-arm64:master
+    ports:
+      - "48090:48090"
+    container_name: edgex-sys-mgmt-agent
+    hostname: edgex-sys-mgmt-agent
+    networks:
+      - edgex-network
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    depends_on:
+      - logging
+
+  notifications:
+    image: nexus3.edgexfoundry.org:10004/docker-support-notifications-go-arm64:master
+    ports:
+      - "48060:48060"
+    container_name: edgex-support-notifications
+    hostname: edgex-support-notifications
+    networks:
+      - edgex-network
+    volumes:
+      - vault-config:/vault/config
+      - /tmp/edgex/secrets/ca/ca.pem:/tmp/edgex/secrets/ca/ca.pem:ro
+    depends_on:
+      - logging
+
+  metadata:
+    image: nexus3.edgexfoundry.org:10004/docker-core-metadata-go-arm64:master
+    ports:
+      - "48081:48081"
+    container_name: edgex-core-metadata
+    hostname: edgex-core-metadata
+    networks:
+      - edgex-network
+    volumes:
+      - vault-config:/vault/config
+      - /tmp/edgex/secrets/ca/ca.pem:/tmp/edgex/secrets/ca/ca.pem:ro
+    depends_on:
+      - logging
+
+  data:
+    image: nexus3.edgexfoundry.org:10004/docker-core-data-go-arm64:master
+    ports:
+      - "48080:48080"
+      - "5563:5563"
+    container_name: edgex-core-data
+    hostname: edgex-core-data
+    networks:
+      - edgex-network
+    volumes:
+      - vault-config:/vault/config
+      - /tmp/edgex/secrets/ca/ca.pem:/tmp/edgex/secrets/ca/ca.pem:ro
+    depends_on:
+      - logging
+
+  command:
+    image: nexus3.edgexfoundry.org:10004/docker-core-command-go-arm64:master
+    ports:
+      - "48082:48082"
+    container_name: edgex-core-command
+    hostname: edgex-core-command
+    networks:
+      - edgex-network
+    volumes:
+      - vault-config:/vault/config
+      - /tmp/edgex/secrets/ca/ca.pem:/tmp/edgex/secrets/ca/ca.pem:ro
+    depends_on:
+      - metadata
+
+  scheduler:
+    image: nexus3.edgexfoundry.org:10004/docker-support-scheduler-go-arm64:master
+    ports:
+      - "48085:48085"
+    container_name: edgex-support-scheduler
+    hostname: edgex-support-scheduler
+    networks:
+      - edgex-network
+    volumes:
+      - vault-config:/vault/config
+      - /tmp/edgex/secrets/ca/ca.pem:/tmp/edgex/secrets/ca/ca.pem:ro
+    depends_on:
+      - metadata
+
+  app-service-rules:
+    image: nexus3.edgexfoundry.org:10004/docker-app-service-configurable-arm64:latest
+    ports:
+      - "48100:48100"
+    container_name: edgex-app-service-configurable-rules
```

Comment:
Need to add temporarily flag `skipVersionCheck=true` to avoid blackbox testing failure on app-service-configurable.
See this: https://github.com/edgexfoundry/blackbox-testing/blob/master/docker-compose.yml#L489-L490

Reply:
Added app-service-configurable in PR [edgexfoundry/blackbox-testing#350](https://github.com/edgexfoundry/blackbox-testing/pull/350), because it seems it's only for blackbox testing.

Reply:
ok